bin/e2e-ci

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# As discussed in the engineering roundtable 2021/01/25, we will revisit cypress-rails knapsack pro integration
+# for re-enabling parallelization of e2e tests later.
+RAILS_ENV=test bundle exec rake cypress:run
```

Comment:
Can you please add the missing newlines GH is highlighting here and in a couple of other files?